.github/workflows/analyze-build-debug.yaml

```diff
@@ -13,6 +13,10 @@ jobs:
     name: Flutter analyze, build
     runs-on: ubuntu-latest
     steps:
+      - name: Save state
```

Comment:
Remove the next 2 steps. echoing random values on CI is not a good practice especially from a security PoV.